src/components/Input/Input.tsx

```diff
@@ -1,5 +1,7 @@
+'use client';
+
```

Comment:
Please, double check if `use client` is really needed here. This stays true for other general components